prql-compiler/Cargo.toml

```diff
@@ -73,3 +73,6 @@ exactly = 1
 file = "../web/book/src/language-features/target.md"
 replace = 'prql version:"{{version}}"'
 search = 'prql version:"[\d.]+"'
+
+[features]
+test-external-dbs = []
```

Comment:
Excellent.

We can also put the external features behind the feature, so we don't need to compile them locally; i.e. something like:

```toml
[dependencies]

[target.'cfg(not(target_family="wasm"))'.dev-dependencies]
chrono = {version = "0.4", features = [], default-features = false}
criterion = "0.4.0"
tokio = {version = "1", features = ["full"]}
tokio-util = { version = "0.7", features = ["compat"] }
pretty_assertions = "1.3.0"
csv = "1.2"
postgres = { version = "0.19.3", optional = true }
mysql = { version = "23", optional = true }
tiberius = { version = "0.12", features = ["sql-browser-tokio", "bigdecimal", "time"], optional = true }
rusqlite = { version = "0.29.0", features = ["bundled", "csvtab"] }
pg_bigdecimal = { version = "0.1", optional = true }

[features]
test-external-dbs = [
    "postgres",
    "mysql",
    "tiberius",
    "pg_bigdecimal",
]
```

Reply:
that doesn't work because dev-dependencies cannot be optional

Reply:
Ah, sorry!